action.yaml

```diff
@@ -66,4 +70,4 @@ runs:
     using: 'docker'
     image: 'Dockerfile'
     entrypoint: '/deploy.sh'
-    post-entrypoint: '/cleanup.sh'
+    post-entrypoint: '/cleanup.sh'
```

Comment:
May we keep the newline at end of file? Looks stripped now.

Reply:
Will sort this 👍